CMakeLists.txt

```diff
@@ -155,7 +155,7 @@ endif()
 # =============================================================================
 # Setup Doxygen documentation
 # =============================================================================
-find_package(Doxygen QUIET)
+find_package(Doxygen)
```

Comment:
Just a comment: many non-developers find_packages were done with QUIET. In the future I wonder if we have explicit option like `NRN_ENABLE_DOC` and that's where we do such stuff.

Reply:
Forgot to revert after debugging.
Not a big fan of adding another option, on the other would be nice to have some feedback for building docs. We can discuss it at a later time.